Configuration/Geometry/python/GeometryDD4hepExtended2026D49_cff.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+
+#
+# Geometry master configuration
+#
+# Ideal geometry, needed for simulation
+DDDetectorESProducer = cms.ESSource("DDDetectorESProducer",
+                                    confGeomXMLFiles = cms.FileInPath('Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2026D49.xml'),
+                                    appendToDataLabel = cms.string('')
+)
+
+DDSpecParRegistryESProducer = cms.ESProducer("DDSpecParRegistryESProducer",
+                                             appendToDataLabel = cms.string('')
+)
+
+DDVectorRegistryESProducer = cms.ESProducer("DDVectorRegistryESProducer",
+                                            appendToDataLabel = cms.string(''))
+
+DDCompactViewESProducer = cms.ESProducer("DDCompactViewESProducer",
+                                         appendToDataLabel = cms.string('')
+)
```

Comment:
This seems to be the main difference vs. the DDD setup, replacing the line:
```python
from Geometry.CMSCommonData.cmsExtendedGeometry2026D49XML_cfi import *
```
Should we standardize to put all of this in a corresponding file like `Geometry.CMSCommonData.cmsExtendedGeometry2026D49XMLDD4hep_cfi`?

It would be even better if there were some way to do this with Eras... then the same `XML_cfi` file could be used for both (with the `DD4hep` special workflow already defined in the upgrade matrix).

@ianna @makortel any thoughts?

Reply:
I think we should wait a while before things stabilizer. We are in the process of getting the Phase2 workflow to start working - it is not there yet. We are still struggling for the Run3 scenario.

Reply:
> It would be even better if there were some way to do this with Eras...

The way to customize the set of ESProducers with Modifiers is to put the ESProducers into a Task and customize the Task with the Modifier.

Admittedly that would imply a larger change in how geometry ESProducers are loaded in configurations (the geometry Task would have to be added to the Schedule/Path/EndPath via the usual means).

Reply:
I was looking at the possibility to have Phase2 test workflows, and geometry configurations automatically built bu the generator script, and I had a similar idea as @kpedro88 , at least for the 3 latest items in this comment it would make sense to have a single ```_cff``` fragment to include, as they are common everywhere.